packages/sdk/src/types.ts

```diff
@@ -39,7 +44,7 @@ export interface Pool {
     chainId: number;
     address: Address;
     amm: SupportedAmm;
-    fee: number;
+    fee: number | null;
```

Comment:
Wait, why is this nullable now?

Reply:
This was actually always nullable but the type was wrong, since we are doing `fee: toNumberOrNull(backendCampaign.pool.fee)`; can the fee actually be null on the backend API response?